examples/stretched_cluster/variables.tf

```diff
@@ -0,0 +1,61 @@
+variable "api_token" {
+  description = "API token used to authenticate when calling the VMware Cloud Services API."
+  default = ""
+}
+
+variable "org_id" {
+  description = "Organization Identifier."
+  default = ""
+}
+
+variable "aws_account_number" {
+  description = "The AWS account number."
+  default     = ""
+}
+
+variable "sddc_name"{
+  description = "Name of SDDC."
+  default = "sddc-test"
+}
+
+variable "sddc_region" {
+  description = "The AWS  or VMC specific region."
+  default     = "us-west-2"
+}
+
+variable "vpc_cidr" {
+  description = "SDDC management network CIDR. Only prefix of 16, 20 and 23 are supported."
+  default     = ""
+}
+
+variable "vxlan_subnet" {
```

Comment:
Is this going to hold 2 subnets?

Reply:
customer_subnet_ids is the field used to specify two subnets.